Dockerfile

```diff
@@ -1,21 +1,25 @@
-FROM maven:3.5.3-jdk-8 AS build
+FROM maven:3.6.0-jdk-11 AS build
 
 # Add pom.xml and install dependencies
 COPY pom.xml /build/pom.xml
 WORKDIR /build
-RUN mvn clean
+RUN mvn -B dependency:go-offline
 
 # Add source code and (by default) build the jar
 COPY src /build/src
-ARG BUILD_COMMAND="mvn verify"
-RUN ${BUILD_COMMAND}
+RUN mvn -B clean package -Dmaven.test.skip=true
```

Comment:
Probably remove the `-Dmaven.test.skip=true` part.  I'm not sure if `nldi-crawler` has integration tests it needs to run, but probably best to have them exercised.